link/src/main/java/com/stripe/android/link/ui/signup/SignUpScreen.kt

```diff
@@ -159,7 +161,9 @@ private fun EmailCollectionSection(
 }
 
 @Composable
-private fun PhoneCollectionSection() {
+private fun PhoneCollectionSection(
```

Comment:
Can we make this consistent with our other elements with section and elements so that it is reusable across our UI?